File: src/main/resources/static/app/models/entities/buildingVm.js
///<reference path="workVm.ts"/>
/**
 * Created by Hassan on 11/18/2016.
 */
var BuildingVm = (function () {
    function BuildingVm(id, name, works) {
        this.setup();
        if (id != undefined)
            this.id = id;
        if (name != undefined)
            this.name = name;
        if (works != undefined)
            this.works = works;
    }
    BuildingVm.prototype.setup = function () {
        this.id = 0;
        this.name = null;
        this.works = [];
    };
    return BuildingVm;
}());

File: src/main/java/hassan/personnel/managment/models/vm/BuildingVm.java
package hassan.personnel.managment.models.vm;

import java.util.List;

/**
 * Created by Hassan on 11/21/2016.
 */
public class BuildingVm {
    private int id;

    private String name;

    private List<WorkVm> works;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public List<WorkVm> getWorks() {
        return works;
    }

    public void setWorks(List<WorkVm> works) {
        this.works = works;
    }
}
